src/js/apps/patients/worklist/filters_app.js

```diff
@@ -111,6 +117,37 @@ export default App.extend({
 
     this.showChildView('owner', ownerFilter);
   },
+  showDateFilterView() {
+    const dateFilter = this.showChildView('date', new DateFilterView({
```

Comment:
I'm now leaning toward this being a component that has it's own state where this app is listening to state events so that it can make a new filtered request vs re-rendering and passing options.